webrtc.html

```diff
@@ -3782,8 +3805,8 @@
                 then perform the following checks:</p>
                 <ul>
                   <li>
-                    <p>If <var>t</var>'s <a data-for=
-                    "RTCRtpTransceiver">direction</a> is "sendrecv" or
+                    <p>If <var>t</var>'s [[<a href=
+                    "#direction-slot">Direction</a>]] slot is "sendrecv" or
```

Comment:
`inactive` doesn't need explicit handling here. This step is just to make sure an "a=msid" line gets added. If you set the direction to "inactive" and it was previously something else, one of the next two bullets will return `true`:

>- If connection's currentLocalDescription if of type "offer", and the direction of the associated m= section in neither the offer nor answer matches t's direction, return "true".
>- If connection's currentLocalDescription if of type "answer", and the direction of the associated m= section in the answer does not match t's direction intersected with the offered direction (as described in [JSEP] (section 5.3.1.)), return "true".